design/ceph/node-loss-rbd-rwo.md

```diff
@@ -0,0 +1,68 @@
+# Recover RBD RWO Pod in case of Node Loss
+
+## Goal
+
+Faster RBD RWO recovery in case of node loss.
+
+## Problem
+
+When a  node is lost where a pod is running with the RBD RWO volume is mounted. The volume cannot be mounted on another node since it could cause corruption if two clients are writing to the same volume.
```

Comment:
```suggestion
When a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. If two clients are write to the same volume it could cause corruption. The node must be guaranteed to be down before the volume can be mounted on another node.
```